src/main/resources/config/Automation_ElevateAPI_QA.properties

```diff
@@ -3,7 +3,13 @@ elevate.qa.api.base.url=https://project-qa.elevate-apis.shikshalokam.org/
 elevate.login.endpoint=user/v1/account/login
 elevate.login.contentcreator=manju@guerrillamail.info
 elevate.login.contentcreator.password=Password@123
-sl.capture.screenshot.on.failure=false
```

Comment:
sl.capture.screenshot.on.failure=false  why this is removed . without this Extended report wont generate

Reply:
sorry this might be a miss from my side